### ISSUE TITLE
Usage of the thermalprinter lib

### ISSUE THREAD

**Issue by xseignard** [JasperP/Thermal-Printer-Story#1]
Hello!

I really like your usage of the thermalprinter lib. Although you say that you roasted 2 printers, if you have any inputs to give me in order to see what was the problem, let me know!

I see that you use johnny-five and an arduino, but the lib is designed to be used without arduino, so if you could give me some insights on how you did it it would be great!

Regards

Xavier

**Comment by fabiantheblind** [comment]
@xseignard 
I think The process.exit in line 122 creates a problem 

https://github.com/JasperP/Thermal-Printer-Story/blob/master/index.js#L122

**Comment by xseignard** [comment]
@fabiantheblind it will terminate the program, you can remove it

**Comment by fabiantheblind** [comment]
Yes I know. But this is @JasperP Job. :smile: I just stumbled over your issue

**Comment by JasperP** [comment]
Hey Xavier. I am glad you like my usage of the thermal printer. I stumbled over the same issues you posted in that [Forum](http://forums.adafruit.com/viewtopic.php?f=19&t=54958). I had to configure the heating time, -interval, and -dots to get decent images instead of weird characters. 
However I still don't know why the the printer stopped working. Maybe I didn't terminate the process properly. The heatings dots where fired but the paper roll, didn't move further. So that issue roasted my printer.
To the library and arduino: 
I did not use the Johnny-Five library, it was just made in the context of a Johnny-Five workshop. Therefore I only used the serialport library. I've used the Arduino Standard Firmata and used the TX and RX pins on the Arduino Uno.
I hope I could give you some insights, though I don't know the reasons for the problems I had.
Nevertheless thanks for the library! It made my little project at least realizable and please keep working on it :)

Kind regards
Jasper
